script/gitlabci/job_cuda11.5.yml

```diff
@@ -100,3 +100,15 @@ linux_nvcc-11.5_clang-12_release:
     ALPAKA_CI_CMAKE_VER: 3.19.7
     CMAKE_CUDA_COMPILER: nvcc
   stage: stageRun1
+
+# nvhpc
+linux_nvhpc-21.11_cuda-11.5_oacc:
+  extends: .base_oacc_nvhpc
+  variables:
+    ALPAKA_CI_UBUNTU_VER: "20.04"
+    ALPAKA_CI_CUDA_VERSION: "11.5"
+    ALPAKA_CI_NVHPC_VER: "21.11"
+    CMAKE_BUILD_TYPE: ""
```

Comment:
Is there a reason we're not setting the `CMAKE_BUILD_TYPE`?

Reply:
We are actually explicitly setting it to no build type. This is because all cmake build types add some `-O` and/or `-g` flags. Passing `-g` or any `-O` (other than `-O1`, I think) to nvc++ breaks the generated code for some tests.

Note, that this backend is highly not production ready. The compiler flags set in the Ci (should) work. If you change any, your have a high change of running into some compiler bug.

Reply:
We are actually explicitly setting it to no build type. This is because all cmake build types add some `-O` and/or `-g` flags. Passing `-g` or any `-O` (other than `-O1`, I think) to nvc++ breaks the generated code for some tests.

Note, that this backend is highly not production ready. The compiler flags set in the Ci (should) work. If you change any, your have a high change of running into some compiler bug.